test/Microsoft.Azure.ServiceBus.UnitTests/QueueClientTests.cs

```diff
@@ -133,5 +133,21 @@ public sealed class QueueClientTests : SenderReceiverClientTestBase
                 await queueClient.CloseAsync();
             }
         }
+
+        [Theory]
+        [MemberData(nameof(TestPermutations))]
+        [DisplayTestMethodName]
+        async Task ServerWaitTimeoutTest(string queueName, int messageCount = 1)
```

Comment:
`ServerWaitTimeoutTest` doesn't really describe what the outcome of the test is or what's expected. Would rather see a name that implies what the outcome is. E.g. `Should_perform_X`